pkg/processor/eventsource/http/eventsource.go

```diff
@@ -101,8 +104,6 @@ func (h *http) requestHandler(ctx *fasthttp.RequestCtx) {
 	var bufferLogger *nucliozap.BufferLogger
 
 	// attach the context to the event
-	h.event.ctx = ctx
```

Comment:
Wow, what an oversight. 
Still, I prefer not to introduce sync.Pools. They only reduce allocations, they don't eliminate them (+ introduce locking). Here we need exactly `# of worker` events. I have an idea to implement this without locking / allocation, need to see how to make it future proof

Reply:
Changed to use a fixed pool implementation.